src/components/ContentHighlights/data/constants.js

```diff
@@ -188,6 +188,7 @@ export const TEST_COURSE_HIGHLIGHTS_DATA = [
         content_key: 'edX+DemoX',
         title: 'Math',
         card_image_url: 'https://picsum.photos/360/200',
+        course_run_statuses: ['archived'],
```

Comment:
Would put 'archived' and 'published' into their own constants since they're used repeatedly in the code.